user-commands.go

```diff
@@ -631,15 +631,15 @@ const (
 	AccessKeyListAll        = "all"
 )
 
-// ListAccessKeysLDAPV2 - list service accounts belonging to the given users or all users
-func (adm *AdminClient) ListAccessKeysLDAPv2(ctx context.Context, userDNs []string, listType string, all bool) (map[string]ListAccessKeysLDAPResp, error) {
-	if len(userDNs) > 0 && all {
+// ListAccessKeysLDAPv2 - list service accounts belonging to the given users or all users
+func (adm *AdminClient) ListAccessKeysLDAPv2(ctx context.Context, users []string, listType string, all bool) (map[string]ListAccessKeysLDAPResp, error) {
```

Comment:
```suggestion
func (adm *AdminClient) ListAccessKeysLDAPBulk(ctx context.Context, users []string, listType string, all bool) (map[string]ListAccessKeysLDAPResp, error) {
```

keep the name appropriate to what it does.